src/mopi/run.py

```diff
@@ -54,14 +55,20 @@ def run(
             plugins=[OutputAnalyserPlugin()] + logger_plugins,
         )
 
-        store = runner.run()
+        store, pipeline = runner.run(pure_inference)
```

Comment:
runner should return pipeline, because we are deepcopying the pipeline so ``experiment.pipeline != pipeline``.